menu/cakes/cakeA.html

```diff
@@ -0,0 +1,409 @@
+                    <option value="cake" selected>Cake</option>
+                    <option value="cupcakes">Cupcakes</option>
+                  </select>
+                </label>
+            
+                <label class="cakeSize" style="display: none;">
+                  Size of the cake
+                  <select class="cakeSizeSelect" name="cake size">
+                    <option value="">- Choose one -</option>
+                    <option value="4in">4-inch (8 servings)</option>
+                    <option value="6in">6-inch (12 servings)</option>
+                    <option value="8in">8-inch (24 servings)</option>
+                    <option value="10in">10-inch (34 servings)</option>
+                  </select>
+                </label>
+            
+                <label class="cupcakesQuantity" style="display: none;">
+                  Quantity of the cupcakes
+                  <select class="cupcakesQuantitySelect" name="cupcakes quantity">
+                    <option value="">- Choose one -</option>
+                    <option value="6">6</option>
+                    <option value="12">12</option>
+                  </select>
+                </label>
+            
+                <label>
+                  When do you need the cake by?
+                  <input type="date" name="date" id="orderNeededDate" required>
+                </label>
+            
+                <label>
+                  Add a Custom Inscription? <i>Limit 40 chars</i> <!-- #TODO - ADD CHARACTER LIMIT -->
+                  <input type="text" name="event" id="event-type" placeholder="leave blank if N/A" maxlength="40">
+                </label>
+            
+                <!-- <label class="cakeFlavors">
+                  Cake Flavor
+                  <select name="cake flavor" required>
+                    <option value="">- Choose one -</option>
+                    <option value="1">Vanilla</option>
+                    <option value="2">Chocolate</option>
+                  </select>
+                </label> -->
+            
+                <!-- <label class="frostingFlavors">
+                  Frosting Flavor
+                  <select name="frosting flavor" required>
+                    <option value="">- Choose one -</option>
+                    <option value="1">Vanilla</option>
+                    <option value="2">Chocolate</option>
+                  </select>
+                </label> -->
+            
+                <label class="dietRestrictions">
+                  Please list any allergies or dietary restrictions
+                  <input type="text" name="dietRestrictions" placeholder="eggs, soy, wheat, etc. (leave blank if n/a)">
+                </label>
+            
+                <label class="colorPalette">
+                  Color Palette
+                  <div class="color-palette"><input type="radio" id="Bright" name="colorPalette" value="Bright"><label for="Bright">Bright</label>
```

Comment:
we may not need color palette options in this page since these cakes are not custom orders. But they looks good, let's put them in the deprecated `orders.html` so that we can use it in the future if Amanda decides to use it.

Reply:
great, confirming I made this fix

Reply:
I kept color palette on cupcake and cookies. depending on how specific the type of cupcake or cookie, this may not be necessary